nixos/modules/services/networking/wireguard.nix

```diff
@@ -137,6 +137,32 @@ let
         See [documentation](https://www.wireguard.com/netns/).
         '';
       };
+
+      fwMark = mkOption {
+        default = null;
+        type = with types; nullOr string;
+        example = "0x6e6978";
+        description = lib.mdDoc ''Mark all wireguard packets originating from
```

Comment:
```suggestion
        description = lib.mdDoc ''
          Mark all wireguard packets originating from
```

Reply:
yeah sounds good, should I redo the commits or just press "commit changes" in the github ui?

Reply:
alright, enough time passed that I decided on my own. some inconsistencies in documentation for some other wireguard options bothered me enough to add a commit to bring these in line with the rest.